code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -246,3 +247,27 @@
 
 /obj/item/clothing/suit/armor/reactive/table/emp_act()
 	return
+
+/obj/item/clothing/suit/armor/reactive/vortex
+	name = "reactive vortex armor"
+	desc = "An experimental suit of armor that uses the power of black holes to stop any attacker."
+
+/obj/item/clothing/suit/armor/reactive/vortex/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(!active)
+		return FALSE
+	if(prob(hit_reaction_chance))
+		if(world.time < reactivearmor_cooldown)
+			owner.visible_message("<span class='danger'>The black hole generators on [owner]'s reactive vortex armor are still recharging! The armor merely emits some sparks.</span>")
+			return
+		var/turf/T = get_turf(owner)
+		for(var/mob/living/A in range(T, 7))
+			visible_message("<span class='notice'>[owner] starts cock-voring [A]!</span>")
+			A.say("NOOOO! HELP ME!!")
+			A.Paralyze(50)
+			if(do_after(owner, 50, TRUE, A))
+				A.say("AAAAAAAAAAAAAAAAAAAA!!")
+				visible_message("<span class='notice'>[A] is sucked into [owner]'s cock, instantly killing them!</span>")
+				A.gib(null, null, TRUE, TRUE)
+		owner.visible_message("<span class='danger'>[src] blocks [attack_text], cock voring the attackers!</span>")
```

Comment:
```suggestion
		owner.visible_message("<span class='danger'>[src] blocks [attack_text], cock-voring the attackers!</span>")
```

Reply:
NO THE MAINTAINERS WILL SEE DELETE

Reply:
also, if they're voring them, why not move A inside of owner (instead of gibbing A)?

Reply:
based

Reply:
![image](https://user-images.githubusercontent.com/40974010/82791353-7196b400-9e22-11ea-8c9c-b84e806e6034.png)